test/e2e_test.go

```diff
@@ -165,18 +194,21 @@ var _ = Describe("e2e", func() {
 				resp, err := channel.CreateChannel(osnURL, block, caCertPool, tlsClientCert)
 				Expect(err).NotTo(HaveOccurred())
 				Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
-
+				fmt.Println("Channel creation")
+				<-block_channel
```

Comment:
the logic here is to use a channel and fetch a block from channel.
but it seems won't work.
@bestbeforetoday , is it due to the error is no reported in the go routine below? as a part of Events()
```
	go func() {
		defer close(results)

		for {
			response, err := eventsClient.Recv()
			result := response.GetBlock()
			if err != nil || result == nil {
				return
			}

			results <- result
		}
	}()
```